styles/gmail/catppuccin.user.css

```
@@ -2,7 +2,7 @@
 @name Gmail Catppuccin
 @namespace github.com/catppuccin/userstyles/styles/gmail
 @homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/gmail
-@version 0.1.4
+@version 0.1.5
```

```suggestion
@version 0.2.0
```

Idk versioning doesn't matter much but this is a bunch of good fixes and improvements so it deserves it :)